src/entities/pools/gyro3/gyro3Math.ts

```diff
@@ -0,0 +1,254 @@
+import {
+    _INVARIANT_MIN_ITERATIONS,
+    _INVARIANT_SHRINKING_FACTOR_PER_STEP,
+} from './constants';
+import { _safeLargePow3ADown } from './helpers';
+import { MathGyro, ONE } from '../../../utils/gyroHelpers/math';
```

Comment:
Switch any `ONE` consts to use the global `WAD`.`ONE` can be confusing because it's still not clear which scale is being used